scripts/exregional_run_prepstart.sh

```diff
@@ -224,6 +224,9 @@ if [ ${BKTYPE} -eq 1 ] ; then  # cold start, use prepare cold strat initial file
       cp_vrfy ${bkpath}/gfs_ctrl.nc gfs_ctrl.nc        
       cp_vrfy ${bkpath}/gfs_data.tile7.halo0.nc gfs_data.tile7.halo0.nc        
       cp_vrfy ${bkpath}/sfc_data.tile7.halo0.nc sfc_data.tile7.halo0.nc        
+      ln_vrfy -s ${bkpath}/gfs_bndy.tile7.000.nc bk_gfs_bndy.tile7.000.nc
+      ln_vrfy -s ${bkpath}/gfs_data.tile7.halo0.nc bk_gfs_data.tile7.halo0.nc
+      ln_vrfy -s ${bkpath}/sfc_data.tile7.halo0.nc bk_sfc_data.tile7.halo0.nc
```

Comment:
Does "ln_vrfy" works for the purpose to "**save background in fcst/INPUT**"?

Reply:
Yes, we only need to kwon where the background come from.